framework/include/vectorpostprocessors/LeastSquaresFit.h

```diff
@@ -69,6 +69,12 @@ class LeastSquaresFit :
   /// The number of samples to be taken
   unsigned int _num_samples;
 
+  /// Values used to scale and or shift x and y data
+  Real _x_scale;
+  Real _x_shift;
+  Real _y_scale;
+  Real _y_shift;
+
```

Comment:
Make those `const`. Use `///@{ ... ///@}` to make the Doxygen comment apply to all for variables.